server/src/main/scala/actors/serializers/FlightMessageConversion.scala

```diff
@@ -145,7 +148,11 @@ object FlightMessageConversion {
       carrierScheduled = apiFlight.CarrierScheduled,
       apiPax = apiFlight.ApiPax,
       redListPax = apiFlight.RedListPax,
-      scheduledDeparture = apiFlight.ScheduledDeparture
+      scheduledDeparture = apiFlight.ScheduledDeparture,
+      totalPax = apiFlight.TotalPax.map(tp => uk.gov.homeoffice.drt.protobuf.messages.FlightsMessage.TotalPaxSource(
```

Comment:
Might be nice to extract a function to convert TotalPax into a TotalPaxSource message